owl/EASE-SAY.owl

```diff
@@ -183,23 +431,17 @@ Note that in a semantic sense such clauses always request information, but in a
     
 
 
-    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#LexicalObject -->
-
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#LexicalObject">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#LinguisticObject"/>
-        <owl:disjointWith rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#PhrasalObject"/>
-        <rdfs:comment>A linguistic object that contains information which would be linguistically expressed as a lexical item (loosely speaking, a word).</rdfs:comment>
-        <rdfs:label>Lexical object</rdfs:label>
-    </owl:Class>
-    
-
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#LocatumRole -->
 
-    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#LinguisticObject -->
-
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#LinguisticObject">
-        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#InformationObject"/>
-        <rdfs:comment>An information bearing object that is used to analyze some linguistic production.</rdfs:comment>
-        <rdfs:label>Linguistic object</rdfs:label>
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#LocatumRole">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#classifies"/>
+                <owl:someValuesFrom rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalObject"/>
```

Comment:
a role can exist without ever being manifested, so universal restriction should be used instead